src/js/wrappers/style.scss

```diff
@@ -1,4 +1,6 @@
 .input-wrapper-container {
+  display: flex;
+  flex-direction: column;
```

Comment:
this feels somewhat dangerous - doesn't it break the positioning of already existing fields?

Reply:
when adding all these new styles I specifically tested all other fields and everything should be fine, do you have any specific concerns why you think it would be dangerous?